IN-CORE/pyincore
# Release 1.1.0 (#39)

* add test script

* change cease to closure

* Delete tests/pyincore/analyses/businessclosure directory

* removed deprecated methods and classes

* removed outdated test and modified  building period method

* Add Lisa Wang fixes.

* removed files and commented code

* #7: Create Repair Curve class to be used with residential recovery analysis

* fixed pep8 issues (#21)

* fixed pep8 issues

* Pyincore util to convert population dislocation output for heatmap #3 (#5)

* add utility function to save as shapefile

* add filter

* write a test

* fix bug

* add snippet of code to test util and upload

* Added conda recipe (#18)

* Added conda recipe

* changed python version to 3.8

* #6: IncoreClient will show the error messages that the api returns. Pytests added (#16)

updated client error messages returned

* added github action for automatic docs build (#24)

* added github action for automatic docs build

* changed the version info in master branch

* modified based on comment

* renamed the file

* #26: run pyincore pytests (#33)

* #26: run pyincore pytests

* #26: Added cache to test performance

* #26: another push to test performance difference

* #26: Move caching to correct place

* #26:another test

* #26: test without mamba

* #26: re-enable mamba

* 34 add percentage in utils cge output to json (#36)

* add percentages, refactor test script, pep8

* joplin pd files

* #28 add joplin building functionality recovery time model (#30)

* restoration model,documentation and test script

* add global target functionality level

* add utility with mean and dev, add numpy arrays, filter no hazard

* FL mapping to Limit states, filter by archetype and chain in tests

* pep8 changes

* Need to pass the dataset type as a parameter to from dataframe method #8 (#35)

* add data_type parameter to pass in

* put correct dataset type to set_result

* add test

* fix bug and add one example test on epf

* remove test code

* add to test_format.py

* fix format

* #9 convert pd outputs to json (#27)

* init functions with test jsons

* ad dislocation by housing

* add input path and output files

* add test script

* expand jsons with population dislocation keys

* test script and list bug

* race, tenure, total calculations

* pop disl id with hhinc column, improved income json

* improve household code, improved total json

* delete test csv output

* doc modules and csv output to json script

* full test script

* change class name to match the python file name

* change names of keys based on the SC input, add percentages

* add percentages, check total zeroes, improve test script

* except missing category key

* add files to test script

* pep8 changes

* add comma in the list of categories

* merge conflicts in tests

* combine popdisl scripts and tests

* comment out service upload

* fix heatmap filter bug

* remove csv output

* Updated CHANGELOG and set version to 1.1.0

Co-authored-by: ncsa-mo <ondrejce@illinois.edu>
Co-authored-by: Diego Calderon <diegoac2@illinois.edu>
Co-authored-by: Santiago Nunez-Corrales <nunezco2@illinois.edu>
Co-authored-by: Gowtham Naraharisetty <vnarah2@illinois.edu>
Co-authored-by: Diego <45798318+diegoac2@users.noreply.github.com>
Co-authored-by: Chen Wang <cwang138@illinois.edu>
Co-authored-by: YONG WOOK KIM <ywkim312@users.noreply.github.com>
Co-authored-by: Gowtham Naraharisetty <59935416+vnarah2@users.noreply.github.com>
